examples/repl.rs

```diff
@@ -45,6 +43,10 @@ use bdk::sled;
 use bdk::Wallet;
 
 use bdk::blockchain::esplora::EsploraBlockchainConfig;
+use bdk::cli::{WalletOpt, WalletSubCommand};
+use bitcoin::hashes::core::str::FromStr;
```

Comment:
I think you can also just import `std::str::FromStr` and it should work

Reply:
fixed